### DISCUSSION TITLE
PAGView makeSnapshot() 获取到的bitmap无法显示

### DISCUSSION THREAD

**Discussion by xyggs** [Tencent/libpag#1892] [Q&A]
【版本信息】
4.3.33 | Android10
【平台信息】
Android 原生 
【预期的表现】
通过makeSnapshot方法获取到的bitmap 显示到 imageView 上

【实际的情况】
在红米k50使用没问题，但在荣耀9x上、pagView在全屏状态下截图，截出来的bitmap无法显示，将其大小固定在一个较小的值后，截图出来的bitmap 才能显示

【Demo及附件】
见demo MakeSnapshotActivity
[pag截图显示demo.zip](https://github.com/Tencent/libpag/files/13406539/pag.demo.zip)

**Comment by h3r3x3** [comment]
固定较小值是什么意思？ 可以在AndroidStudio里面debug一下生成的Bitmap，看下断点时机的Bitmap能否预览

**Comment by CodeJhF** [comment]
感谢 @h3r3x3 的回复。

因长期无回复，且 libpag 4.3 及更早版本已停止维护（当前仅维护 4.4 / 4.5），先关闭本条。
上报版本：4.3.33。

如果问题在最新版本（4.4 / 4.5）仍能复现，欢迎在本帖回复或新开 issue，并附上 libpag 版本号、平台与系统版本、可复现的 PAG 文件或最小 demo，我们会优先跟进。（stale-close）
